### ISSUE TITLE
Can the the links start going to the new editor?

### ISSUE THREAD

**Issue by jsdanie1** [onurmenal/crm-power-pane#146]
First of all this tool is amazing. I use it daily!!

One request though, the feature for form editor, can there be a button that opens it to the new form Editor? Microsoft for quite awhile has moved stuff to Power Apps. So would like an option to have the form editor go to that.

**Comment by mehmetseckin** [comment]
Yes I feel like this should be the default - we could still keep an option to go to the legacy editor.

**Comment by jsdanie1** [comment]
Hi Did this feature ever get added? I guess I'm confused on all the different versions of Power Pane. I was using one version a week ago, which had this feature, but then Edge flagged it as Malwarebytes one day, so I had to uninstall it, I then reinstalled the Chrome version of this app, which does work. it supposedly has a newer version number, but doesn't have this feature anymore. I assumed maybe the chrome Extension slightly different from the Original Edge version of Power Pane. Ether way could we get this feature added? Been another 2 years. 

i guess i'm confused who actually manages this extension? Its one of the best ones out there, but for some reason its been off and on with the extension store the past year.
